rss/serializers/atom_extensions.py

```diff
@@ -0,0 +1,63 @@
+from feedgen.ext.base import BaseExtension, BaseEntryExtension
+from lxml import etree
+
+
+class ArxivExtension(BaseExtension):
+    def __init__(self):
+        pass
+
+    def extend_atom(self, atom_feed):
+        return atom_feed
+
+    def extend_rss(self, rss_feed):
+        return rss_feed
+
+    def extend_ns(self):
+        return {'arxiv': 'http://arxiv.org/schemas/atom'}
+
+
+class ArxivEntryExtension(BaseEntryExtension):
```

Comment:
I'd suggest adding some more documentation to this class, as well as some of its methods. When we add Travis CI, it will check for this sort of thing as well.

Reply:
I have made a pass through all the current code to add doc to satisfy pydocstyle, and will try to maintain that with upcoming submissions.